catbuffer/schemas/symbol/all_generated.cats

```diff
@@ -0,0 +1,17 @@
+# block
```

Comment:
all.cats should include this along with all state definitions

Reply:
do we actually use `all.cats` anywhere?

Reply:
only in parser testvectors